uservice_ccutter/server.py

```diff
@@ -215,7 +215,7 @@ def create_github_repository(auth, userdict):
         ghub = github3.login(auth["username"], token=auth["password"])
         try:
             ghub.me()
-        except github3.exceptions.AuthenticationFailed:
+        except (github3.exceptions.AuthenticationFailed, AttributeError):
```

Comment:
Use `as` instead of the comma. See try except programming recommendations in PEP-8 https://www.python.org/dev/peps/pep-0008/#programming-recommendations